### PR TITLE
#115 provide the script sample link in "script editor"

### PR DESCRIPTION
**Pull request by GwonGisoo**

### CODE REVIEW

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 1]
```diff
@@ -314,6 +314,7 @@ script.editor.title=\u811a\u672c\u7f16\u8f91\u5668
 script.editor.button.validate=\u9a8c\u8bc1\u811a\u672c
 script.editor.message.overWriteNewer=\u5728\u7cfb\u7edf\u4e2d\u5b58\u5728\u6700\u8fd1\u4fee\u6539\u7684\u811a\u672c\uff0c\u60a8\u8fd8\u8981\u91cd\u5199\u5f53\u524d\u811a\u672c\u5417\uff1f
 script.editor.error.validate=\u9a8c\u8bc1\u811a\u672c\u53d1\u751f\u9519\u8bef\u3002
+script.editor.sample.message=If you want to know how to writing script, click <b>Sample</b> button.<br/>Then you can see the sample script.
```
Comment (GwonGisoo) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: @chengaomin Would you please translate this?

File: ngrinder-controller/src/main/resources/messages_en.properties [hunk 2]
```diff
@@ -318,6 +318,7 @@ script.editor.title=Script Editor
 script.editor.button.validate=Validate Script
 script.editor.message.overWriteNewer=There exists the more recent script in SVN. Would you like overwrite it?
 script.editor.error.validate=Script Validation Error.
+script.editor.sample.message=If you want to know how to writing script, click <b>Sample</b> button.<br/>Then you can see the sample script.
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_en.properties]: If you want to see the sample scripts, click <b>Sample</b> button.

File: ngrinder-controller/src/main/resources/messages_cn.properties (hunk 1 above)
Comment (GwonGisoo) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: @chengaomin Sorry, I ask rude to you. I will resolve myself.

Comment (chengaomin) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: @GwonGisoo That's all right.
script.editor.sample.message=\u70b9\u51fb<b>Sample</b> \u6309\u94ae,\u67e5\u770b\u66f4\u591a\u811a\u672c\u793a\u4f8b

Comment (chengaomin) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: `script.editor.sample.message=\u70b9\u51fb<b>Sample</b> \u6309\u94ae,\u67e5\u770b\u66f4\u591a\u811a\u672c\u793a\u4f8b`

Comment (GwonGisoo) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: @chengaomin Wow!! Thank you very much!!!

File: script-sample/login-web/login.groovy [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        request = new HTTPRequest()
+        test.record(request);
+    }
+ 
+    @BeforeThread
+    public void beforeThread() {
+        grinder.statistics.delayReports=true;
+         
+        // reset to the all cookies
+        HTTPRequest loginReq = new HTTPRequest()
```
Comment (junoyoon) [inline script-sample/login-web/login.groovy]: u don't need to make new req. just reuse request here.

File: script-sample/login-web/login.groovy [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        // do login & save to the login info in cookies 
+        NVPair[] params = [new NVPair("id", "MY_ID"), new NVPair("pw", "MY_PASSWORD")];
+        HTTPResponse res = loginReq.POST("https://login.site.com/login/do", params);
+        cookies = CookieModule.listAllCookies(threadContext)
+    }
+ 
+    @Before
+    public void before() {
+        // set cookies for login state
+        Object threadContext = HTTPPluginControl.getThreadHTTPClientContext()
```
Comment (junoyoon) [inline script-sample/login-web/login.groovy]: def instead of Object.

**Pull request merged**